doc/rtd/topics/analyze.rst

```diff
@@ -265,15 +272,18 @@ to cloud-init startup, and tracking regression.
 
 .. code-block:: shell-session
 
-  $ cloud-init analyze boot
-  -- Most Recent Boot Record --
-      Kernel Started at: 2019-08-29 01:35:37.753790
-      Kernel ended boot at: 2019-08-29 01:35:38.807407
-      Kernel time to boot (seconds): 1.053617000579834
-      Cloud-init activated by systemd at: 2019-08-29 01:35:43.992460
-      Time between Kernel end boot and Cloud-init activation (seconds): 5.185053110122681
-      Cloud-init start: 2019-08-29 08:35:45.867000
-  successful
+    $ cloud-init analyze boot
```

Comment:
Same comment about `Example Output:` line between command and example output.

Reply:
+1